go/vt/sqlparser/parse_test.go

```diff
@@ -1882,6 +1913,30 @@ begin
 	select @s;
 end`,
 			output: "create procedure p1 (in n double, in m double) begin\nset @s = '';\nif n = m then set @s = 'equals';\nelse if n > m then set @s = 'greater';\nelse set @s = 'less';\nend if; set @s = concat('is ', @s, ' than');\nend if;\nset @s = concat(n, ' ', @s, ' ', m, '.');\nselect @s from dual;\nend",
+		}, { // DECLARE statements are only allowed inside of BEGIN/END blocks
+			input: "create procedure p1 () begin\ndeclare cond_name condition for 1002;\nend",
```

Comment:
Embedded newlines are a bit janky, consider doing backtick quoting instead

Reply:
Maybe that looks weird because you can't indent any lines? I don't know try it out

Reply:
I do the backtick quoting in other lines, so I can do it here too.